frontend/src/assets/technicalLabelIcon.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+export function JavaIcon(props: React.SVGProps<SVGSVGElement>) {
```

Comment:
```suggestion
export const JavaIcon = (props: React.SVGProps<SVGSVGElement>) => {
```
svg를 이렇게 사용할 수 있군요!! 좋은 거 알아갑니다!

그런데 이것도 화살표 함수로 가능하지 않나요?

Reply:
icon 사이트에서 그대로 복사해오느라 확인못했네용 🥲